docs/content/manual/pre-release/ha/single-replica-node-down.md

```diff
@@ -20,7 +21,7 @@ https://github.com/longhorn/longhorn/issues/2309
 8. Verify volume attached & accessible by pod (i.e test data is available).
 
 ## Node restart/down scenario with `Pod Deletion Policy When Node is Down` set to `delete-both-statefulset-and-deployment-pod`
-1. Create RWO|RWX volume with replica count = 1 & data locality = enabled|disabled.
+1. Create RWO|RWX volume with replica count = 1 & data locality = enabled|disabled|strict-local.
```

Comment:
@derekbit does local volume support RWX? It seems we don't have a specific limitation, but however that mode is specifically for RWO.

Reply:
Need to add strict-local to the first test case as well.

Reply:
> does local volume support RWX? It seems we don't have a specific limitation, but however that mode is specifically for RWO.

Yes. It is supported.

Reply:
The reason we only test for RWO, because testing the volume with data-locality-strict does not make sense for RWX, because the volume will be just next to the share manager pod instead of the user workload pod.